cdap-ui/app/cdap/components/AbstractWidget/CSVWidget/index.tsx

```diff
@@ -15,19 +15,25 @@
  */
 
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import CSVRow from 'components/AbstractWidget/CSVWidget/CSVRow';
 import ThemeWrapper from 'components/ThemeWrapper';
 import AbstractMultiRowWidget, {
   IMultiRowProps,
 } from 'components/AbstractWidget/AbstractMultiRowWidget';
+import { objectQuery } from 'services/helpers';
+import { WIDGET_PROPS } from '../constants';
 
-interface ICSVWidgetProps extends IMultiRowProps {
-  valuePlaceholder?: string;
+interface ICSVWidgetProps {
```

Comment:
Should this be,
```
interface ICSVWidgetProps extends IMultiRowWidgetProps {
  'value-placeholder'?: string;
}
```
Since the template parameter for `IMultiRowProps` is of type `W` that extends `IMultiRowWidgetProps`